src/unix/linux-syscalls.c

```diff
@@ -323,13 +335,18 @@ int uv__epoll_pwait(int epfd,
                     int timeout,
                     uint64_t sigmask) {
 #if defined(__NR_epoll_pwait)
-  return syscall(__NR_epoll_pwait,
+  int result = syscall(__NR_epoll_pwait,
```

Comment:
Minor style issues: libuv always splits the declaration from the assignment.  Can you make sure to line up the function arguments?